docker/config_template.yaml

```diff
@@ -194,31 +194,31 @@ services:
         {{- end }}
 
 clusterMetadata:
-    enableGlobalDomain: false
+    enableGlobalDomain: {{ default .Env.ENABLE_GLOBAL_DOMAIN "false" }}
     failoverVersionIncrement: 10
-    masterClusterName: "active"
-    currentClusterName: "active"
+    masterClusterName: "master"
+    currentClusterName: "master"
     clusterInformation:
-        active:
+        master:
             enabled: true
             initialFailoverVersion: 0
             rpcName: "cadence-frontend"
             rpcAddress: "127.0.0.1:7933"
 
 dcRedirectionPolicy:
-    policy: "noop"
+    policy: {{ default .Env.DC_REDIRECT_POLICY "selected-apis-forwarding" }}
     toDC: ""
 
 archival:
   history:
-    status: "enabled"
+    status: {{ default .Env.HISTORY_ARCHIVAL "disabled" }}
```

Comment:
@andrewjdawson2016 this doesn't work because of https://github.com/uber/cadence/blob/master/common/service/config/archival.go#L53

Background: a customer need to disable archival, they are running pure docker container using our images. 

Do you have any suggestion? One way is to relax the "validDisable" check, not sure if you like that.
Another way is completely remove the archival section for everyone --- I don't think file base archival is useful here as the docker container will be destroyed. but it's a little riscky.